README.md

```diff
@@ -1153,6 +1214,89 @@ prompt.multi_select("Select drinks?", choices, max: 3)
 # ‣ ⬡ bourbon
 ```
 
+#### 2.6.3.10 `:submit_keys`
+
+This works similar to the [same option for `select`](#2628-submit_keys).
+You can configure which key(s) submit your selection (`:return` by default): 
+
+```ruby
+choices = %w(vodka beer wine whisky bourbon)
+prompt.multi_select("Select drinks?", choices, submit_keys: [:ctrl_s])
+# =>
+# Select drinks? vodka, beer, whisky
+#   ⬢ vodka
+#   ⬢ beer
+#   ⬡ wine
+#   ⬢ whisky
+# ‣ ⬡ bourbon
+```
+
+The user can then press the `Ctrl+S` key to confirm their selection:
+```ruby
+# =>
+# Select drinks? vodka, beer, whisky
+```
+
+Please note that alphanumeric keys are *not* supported.
```

Comment:
In the end I thought I'll give it a quick try and see just how much complexity it might actually add.
Have a look at the last commit, I think in the end it wasn't too bad!